agent_build/package_builders.py

```diff
@@ -1048,6 +1048,15 @@ class K8sPackageBuilder(ContainerPackageBuilder):
     RESULT_IMAGE_NAMES = ["scalyr-k8s-agent"]
 
 
+class K8sWithOpenMetricsMonitorsPackageBuilder(ContainerPackageBuilder):
+    """
+    An image for running the agent on Kubernetes with Kubernetes Open Metrics monitor enabled.
+    """
+
+    PACKAGE_TYPE = constants.PackageType.K8S
+    RESULT_IMAGE_NAMES = ["scalyr-k8s-agent-with-om-monitors"]
```

Comment:
Wouldn't it more helpful if we make public name more descriptive, like `scalyr-k8s-agent-with-om-monitors` -> `scalyr-k8s-agent-with-openmetrics-monitors`. On the other hand, it seems very long now  :grin:

I guess, I also shouldn't exclude that the 'om' may be already widely accepted contraction for openmetrics :)

Reply:
Yeah, not sure if there is an upper limit for Docker image names.

Reply:
I think people, who know what they are looking for, shouldn't have any problems with that naming, so I'm ok with that.